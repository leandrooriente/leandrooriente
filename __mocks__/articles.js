const articles = [{
  slug: 'test-test',
  url: '/test/test',
  title: 'Test',
  author: 'Test',
  lang: 'pt-br',
  date: '10/10/2010',
  thumbnail: 'test.png',
  html: '<p>Test</p>',
}, {
  slug: 'foo-bar',
  url: '/foo/bar',
  title: 'Foo Bar',
  author: 'Foo Bar',
  lang: 'pt-br',
  date: '12/12/2012',
  thumbnail: 'foo.png',
  html: '<p>Foo Bar</p>',
}];

export default articles;
