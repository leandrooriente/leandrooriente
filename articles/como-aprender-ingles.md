title:  "Como aprender inglês"
author: Leandro Oriente
lang: pt-BR
date: 2014-03-09
description: "Tracei uma nova meta de vida. Aprender inglês de verdade em 1 ano. Tenho uma base razoavel a bastante tempo que me permite ler artigos, posts, documentações em inglês e compreender boa parte delas mas que não é o suficiente para conversar por 5 minutos com alguém fluente sem parecer um idiota."
thumbnail: /images/joel-santana-thumb.jpg
---

Tracei uma nova meta de vida. Aprender inglês de verdade em 1 ano. Tenho uma base razoavel a bastante tempo que me permite ler artigos, posts, documentações em inglês e compreender boa parte delas mas que não é o suficiente para conversar por 5 minutos com alguém fluente sem parecer um idiota.


Comecei no início do ano a pesquisar a melhor forma aprender inglês e achei muitas opções diferentes. Vou comentar sobre algumas e mostrar o que tem funcionado comigo.

## O que não funcionou por enquanto

### Cursos de inglês

Como comentei, minha meta é conseguir ter uma base razoável para me comunicar bem o suficiente para trabalhar em equipe ou viver em um país que tenha o inglês como idioma principal em um ano.

A maioria das intituições convencionais tem cursos de 18 meses ou mais de duração. O que estrapola um pouco meu prazo. Além disso eu teria que estudar com algumas dezenas de alunos e sou tímido demais. Provavelmente meu rendimento seria abaixo do esperado.

Outro problema que tenho é em relação ao meu horário. Como o horário na empresa é flexível, gosto de chegar mais tarde e sair mais tarde para evitar pegar trânsito e seria difícil de encaixar o curso nesses intervalos.

### Cursos no exterior

Pesquisei durante um bom tempo por cursos no exterior, mas no final cheguei a conclusão de que o investimento é alto demais. Provavelmente é a forma mais eficiente e rápida de se aprender, já que estudando no curso ou não você estaria em contato com o idioma 24 horas por dia. Mas os valores não são dos mais amigáveis.

### Cursos online

Pesquisei muito pouco a respeito. Procurei por depoimentos reais na internet e não achei muita coisa e a publicidade dos cursos não me agradou muito. Fiquei receoso e descartei. Talvez faça durante um mês no decorrer do ano só para ver como funciona, mas por enquanto não entrou no meu fluxo de estudo.

### Podcasts

Na verdade eu não tive nenhuma experiência com podcasts ainda. Devo começar a escutar alguns na ida e volta do trabalho. Não tenho uma opinião formada sobre, mas já recebi boas indicações.

## O que está funcionando

### Aulas particulares

Estava quase certo de que não seria viável ter aulas particulares com algum professor nativo devido ao custo. Porém depois de algumas pesquisas achei o Jonathan. Ele é britânico e atende na Zona Sul do RJ. Como minha empresa fica em Botafogo, consegui ter aulas dentro da própria empresa com mais dois amigos o que torna o custo mais em conta.

Estou gostando demais e tem funcionado muito bem. Tanto que pretendo dobrar o investimento.

### Aplicativos

Por enquanto só utilizo o Duolingo e estou gostando muito. Ele vai fazendo uns exercícios repetitivos e isso te força a pegar mais vocabulário. No início é bem bobinho mas no decorrer do tempo vai ficando mais avançado. Sem contar que ele vai retirando alguns níveis que você conquistou a um tempo para que possa reforçar o que aprendeu.

A comunidade por trás do game é bem forte também. Caso tenha dúvida em uma questão basta olhar o histórico dela que tem gente explicando detalhadamente a resposta correta.

Não conheço nenhum outro bom aplicativo, mas casos tenha uma sugestão, fique a vontade para comentar.

### Séries e filmes com legenda em inglês

Sempre gostei muito de séries e filmes. Como baixo boa parte deles (ainda não sou um adepto do Netflix) sempre vejo com audio em inglês e legenda em português.

De um tempo pra cá comecei a assistir mudando a legenda para inglês. No início é um pouco estranho, mas com o tempo fui me acostumando. Vez ou outra preciso parar para entender determinada palavra chave ou expressão, mas no geral da para entender bem e sem muitas pausas.

Esse exercício além de divertido, já que assistia os filmes e episódios de uma forma ou de outra é muito eficiente para pegar a pronúncia das palavras.

### Setar como inglês o idioma dos softwares e aplicativos

Já tinha o costume de usar alguns sistemas e aplicativos em inglês, principalmente quando não tinha opção de usar em português ou quando os tutoriais e documentações a respeito eram em inglês.

Agora setei o inglês como idioma primário de qualquer sistema que uso. Inclusive do Sistema Operacional do notebook e do smartphone, conta do google, jogos de vídeo game, aplicativos do celular... Tudo mesmo.

Isso é útil pois ganho vocabulário técnico.

### Sites de bate papo

Usei durante um dia o Omegle (com filtro de porn ligado) e gostei muito do resultado. Conversei durante um tempo com uma egípicia mas sem vídeo. Em breve quero começar a conversar pela webcam mesmo.

Gostei do Omegle justamente por conta do filtro. Quem já usou o Chatroullete sabe que a cada 5 chats, 4 tem alguém se masturbando.

## Conclusão

Considero que estou evoluindo bem rápido e com um investimento bem razoável. Como tenho algumas particularidades como o horário quebrado, timidez e prazo curto procurei por métodos alternativos.

Pretendo juntar uma graninha para passar cerca de 1 mês no final do ano em um país com inglês como lingua oficial. Talvez fazer um curso para concretizar o que aprendi durante o ano. Não tenho como usuar como forma oficial de estudo já que o investimento é grande, mas posso tentar fechar esse primeiro ciclo assim.