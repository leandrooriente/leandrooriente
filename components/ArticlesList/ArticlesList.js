import React from 'react';
import PropTypes from 'prop-types';
import { Flex, Box } from 'grid-styled';
import sortByDate from 'helpers/sortByDate';
import Subtitle from 'components/Subtitle';
import ArticleShape from './propTypes/article';
import ArticleItem from './components/ArticleItem';
import css from './ArticlesList.css';

const ArticlesList = ({ articles }) => {
  if (!articles || !articles.length) {
    return null;
  }

  const sortedByDateArticles = articles.sort(sortByDate);

  return (
    <section className={css.wrapper}>
      <Flex flexDirection={['column', 'row']}>
        <Box w={[1, 1 / 4]}>
          <Subtitle>Articles</Subtitle>
        </Box>
        <Box w={[1, 3 / 4]}>
          {sortedByDateArticles.map(article => (
            <ArticleItem article={article} key={article.id} />
          ))}
        </Box>
      </Flex>
    </section>
  );
};

ArticlesList.propTypes = {
  articles: PropTypes.oneOfType([
    PropTypes.arrayOf(
      PropTypes.shape(ArticleShape)
    ),
    PropTypes.array,
  ]).isRequired,
};

export default ArticlesList;
