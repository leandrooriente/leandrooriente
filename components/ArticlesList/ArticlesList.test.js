import React from 'react';
import { mount, shallow } from 'enzyme';
import articles from 'data/articles';
import ArticlesList from './ArticlesList';

describe('<ArticlesList />', () => {
  it('should render <ArticlesList /> component', () => {
    const wrapper = mount(
      <ArticlesList articles={articles} />
    );

    expect(wrapper).toMatchSnapshot();
  });

  it('should return null when case receive an empty array of articles', () => {
    const wrapper = shallow(
      <ArticlesList articles={[]} />
    );

    expect(wrapper.getElement()).toBe(null);
  });
});
