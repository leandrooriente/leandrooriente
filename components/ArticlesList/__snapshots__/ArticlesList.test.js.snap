// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArticlesList /> should render <ArticlesList /> component 1`] = `
<ArticlesList
  articles={
    Array [
      Object {
        "author": "Leandro Oriente",
        "date": "2016-08-23T22:18:30.000Z",
        "description": "Como chegar ao máximo de performance usando static generators e NGINX",
        "html": null,
        "lang": "pt-BR",
        "slug": "criando-sites-estaticos-extremamente-performaticos",
        "thumbnail": "/images/webperformance-thumb.png",
        "title": "Criando sites estáticos extremamente performáticos",
        "url": "/criando-sites-estaticos-extremamente-performaticos",
      },
      Object {
        "author": "Leandro Oriente",
        "date": "2014-07-29T00:00:00.000Z",
        "description": "Uma dúvida recorrente que vejo em grupos ou que recebo por e-mail é sobre como cobrar por um freela. Particularmente não faço freelas tem muito tempo e não pretendo fazer mais, porém aprendi alguns métodos bem eficazes no período em que fazia e que funcionaram muito bem.",
        "html": null,
        "lang": "pt-BR",
        "slug": "freelancer-como-cobrar-por-um-freelance",
        "thumbnail": "/images/clientes-cobrar-thumb.jpg",
        "title": "Como cobrar por um freelance",
        "url": "/freelancer-como-cobrar-por-um-freelance",
      },
      Object {
        "author": "Leandro Oriente",
        "date": "2014-03-09T00:00:00.000Z",
        "description": "Tracei uma nova meta de vida. Aprender inglês de verdade em 1 ano. Tenho uma base razoavel a bastante tempo que me permite ler artigos, posts, documentações em inglês e compreender boa parte delas mas que não é o suficiente para conversar por 5 minutos com alguém fluente sem parecer um idiota.",
        "html": null,
        "lang": "pt-BR",
        "slug": "como-aprender-ingles",
        "thumbnail": "/images/joel-santana-thumb.jpg",
        "title": "Como aprender inglês",
        "url": "/como-aprender-ingles",
      },
      Object {
        "author": "Leandro Oriente",
        "date": "2014-01-19T00:00:00.000Z",
        "description": "Não consigo clientes porque não tenho portfólio, não tenho portfólio porque não consigo clientes",
        "html": null,
        "lang": "pt-BR",
        "slug": "nao-tenho-portfolio-porque-nao-tenho-clientes",
        "thumbnail": "/images/nao-tenho-portfolio-thumb.jpg",
        "title": "Não consigo clientes porque não tenho portfólio",
        "url": "/nao-tenho-portfolio-porque-nao-tenho-clientes",
      },
      Object {
        "author": "Leandro Oriente",
        "date": "2014-01-12T00:00:00.000Z",
        "description": "Atenção, esse é um post polêmico. Antes de iniciar o processo de leitura, esvazie a mente, procure um lugar tranquilo e preparesse porque a mensagem não é das mais agradáveis.",
        "html": null,
        "lang": "pt-BR",
        "slug": "seu-cliente-precisa-de-um-site",
        "thumbnail": "/images/cliente-nao-precisa-site-thumb.jpg",
        "title": "Meu cliente realmente precisa de um site?",
        "url": "/seu-cliente-precisa-de-um-site",
      },
      Object {
        "author": "Leandro Oriente",
        "date": "2013-10-15T00:00:00.000Z",
        "description": "O foco do post é atender a galera iniciante e encurtar alguns caminhos, porém se você é mais cascudo e quiser colaborar, fique a vontade.",
        "html": null,
        "lang": "pt-BR",
        "slug": "guia-de-estudo-front-end",
        "thumbnail": "/images/guia-comovente-thumb.jpg",
        "title": "Um comovente guia de estudo para desenvolvedores front end",
        "url": "/guia-de-estudo-front-end",
      },
      Object {
        "author": "Leandro Oriente",
        "date": "2013-08-21T00:00:00.000Z",
        "description": "Minha visão de obrigações e diferenciais de desenvolvedores front end e algumas pequenas dicas a respeito.",
        "html": null,
        "lang": "pt-BR",
        "slug": "como-me-tornar-um-desenvolvedor-front-end",
        "thumbnail": "/images/developer-thumb.jpg",
        "title": "Como me tornar um Desenvolvedor Front End",
        "url": "/como-me-tornar-um-desenvolvedor-front-end",
      },
      Object {
        "author": "Leandro Oriente",
        "date": "2013-08-17T00:00:00.000Z",
        "description": "Quero mostrar minha experiência atual com o mercado de trabalho e com o mundo acadêmico para depois apresentar as conclusões que tirei.",
        "html": null,
        "lang": "pt-BR",
        "slug": "faculdade-de-informatica-fazer-ou-nao-fazer",
        "thumbnail": "/images/classroom-thumb.jpg",
        "title": "Faculdade de Informática. Fazer ou não fazer?",
        "url": "/faculdade-de-informatica-fazer-ou-nao-fazer",
      },
    ]
  }
>
  <section
    className="wrapper"
  >
    <Flex
      flexDirection={
        Array [
          "column",
          "row",
        ]
      }
      theme={
        Object {
          "breakpoints": Array [
            "40em",
            "52em",
            "64em",
          ],
          "fontSizes": Array [
            12,
            14,
            16,
            20,
            24,
            32,
            48,
            64,
            96,
            128,
          ],
          "space": Array [
            0,
            4,
            8,
            16,
            32,
            64,
            128,
            256,
            512,
          ],
        }
      }
    >
      <Clean.div
        blacklist={
          Array [
            "m",
            "mt",
            "mr",
            "mb",
            "ml",
            "mx",
            "my",
            "p",
            "pt",
            "pr",
            "pb",
            "pl",
            "px",
            "py",
            "width",
            "w",
            "fontSize",
            "f",
            "color",
            "bg",
            "fontFamily",
            "font",
            "textAlign",
            "align",
            "lineHeight",
            "fontWeight",
            "letterSpacing",
            "display",
            "maxWidth",
            "minWidth",
            "height",
            "maxHeight",
            "minHeight",
            "size",
            "ratio",
            "alignItems",
            "alignContent",
            "justifyContent",
            "justify",
            "flexWrap",
            "wrap",
            "flexBasis",
            "flexDirection",
            "flex",
            "justifySelf",
            "alignSelf",
            "order",
            "gridGap",
            "gridColumnGap",
            "gridRowGap",
            "gridColumn",
            "gridRow",
            "gridAutoFlow",
            "gridAutoColumns",
            "gridAutoRows",
            "gridTemplateColumns",
            "gridTemplateRows",
            "border",
            "borderTop",
            "borderRight",
            "borderBottom",
            "borderLeft",
            "borderColor",
            "borderRadius",
            "boxShadow",
            "background",
            "backgroundImage",
            "bgImage",
            "backgroundSize",
            "bgSize",
            "backgroundPosition",
            "bgPosition",
            "backgroundRepeat",
            "bgRepeat",
            "position",
            "zIndex",
            "top",
            "right",
            "bottom",
            "left",
            "hover",
            "focus",
            "active",
            "disabledStyle",
            "textStyle",
            "colors",
            "buttonStyle",
            "borderWidth",
          ]
        }
        className="sc-bwzfXH jyiNXZ"
        flexDirection={
          Array [
            "column",
            "row",
          ]
        }
        is="div"
        theme={
          Object {
            "breakpoints": Array [
              "40em",
              "52em",
              "64em",
            ],
            "fontSizes": Array [
              12,
              14,
              16,
              20,
              24,
              32,
              48,
              64,
              96,
              128,
            ],
            "space": Array [
              0,
              4,
              8,
              16,
              32,
              64,
              128,
              256,
              512,
            ],
          }
        }
      >
        <div
          className="sc-bwzfXH jyiNXZ"
        >
          <Box
            theme={
              Object {
                "breakpoints": Array [
                  "40em",
                  "52em",
                  "64em",
                ],
                "fontSizes": Array [
                  12,
                  14,
                  16,
                  20,
                  24,
                  32,
                  48,
                  64,
                  96,
                  128,
                ],
                "space": Array [
                  0,
                  4,
                  8,
                  16,
                  32,
                  64,
                  128,
                  256,
                  512,
                ],
              }
            }
            w={
              Array [
                1,
                0.25,
              ]
            }
          >
            <Clean.div
              blacklist={
                Array [
                  "m",
                  "mt",
                  "mr",
                  "mb",
                  "ml",
                  "mx",
                  "my",
                  "p",
                  "pt",
                  "pr",
                  "pb",
                  "pl",
                  "px",
                  "py",
                  "width",
                  "w",
                  "fontSize",
                  "f",
                  "color",
                  "bg",
                  "fontFamily",
                  "font",
                  "textAlign",
                  "align",
                  "lineHeight",
                  "fontWeight",
                  "letterSpacing",
                  "display",
                  "maxWidth",
                  "minWidth",
                  "height",
                  "maxHeight",
                  "minHeight",
                  "size",
                  "ratio",
                  "alignItems",
                  "alignContent",
                  "justifyContent",
                  "justify",
                  "flexWrap",
                  "wrap",
                  "flexBasis",
                  "flexDirection",
                  "flex",
                  "justifySelf",
                  "alignSelf",
                  "order",
                  "gridGap",
                  "gridColumnGap",
                  "gridRowGap",
                  "gridColumn",
                  "gridRow",
                  "gridAutoFlow",
                  "gridAutoColumns",
                  "gridAutoRows",
                  "gridTemplateColumns",
                  "gridTemplateRows",
                  "border",
                  "borderTop",
                  "borderRight",
                  "borderBottom",
                  "borderLeft",
                  "borderColor",
                  "borderRadius",
                  "boxShadow",
                  "background",
                  "backgroundImage",
                  "bgImage",
                  "backgroundSize",
                  "bgSize",
                  "backgroundPosition",
                  "bgPosition",
                  "backgroundRepeat",
                  "bgRepeat",
                  "position",
                  "zIndex",
                  "top",
                  "right",
                  "bottom",
                  "left",
                  "hover",
                  "focus",
                  "active",
                  "disabledStyle",
                  "textStyle",
                  "colors",
                  "buttonStyle",
                  "borderWidth",
                ]
              }
              className="sc-bdVaJa eyHDoe"
              is="div"
              theme={
                Object {
                  "breakpoints": Array [
                    "40em",
                    "52em",
                    "64em",
                  ],
                  "fontSizes": Array [
                    12,
                    14,
                    16,
                    20,
                    24,
                    32,
                    48,
                    64,
                    96,
                    128,
                  ],
                  "space": Array [
                    0,
                    4,
                    8,
                    16,
                    32,
                    64,
                    128,
                    256,
                    512,
                  ],
                }
              }
              w={
                Array [
                  1,
                  0.25,
                ]
              }
            >
              <div
                className="sc-bdVaJa eyHDoe"
              >
                <Subtitle>
                  <h3
                    className="title"
                  >
                    Articles
                  </h3>
                </Subtitle>
              </div>
            </Clean.div>
          </Box>
          <Box
            theme={
              Object {
                "breakpoints": Array [
                  "40em",
                  "52em",
                  "64em",
                ],
                "fontSizes": Array [
                  12,
                  14,
                  16,
                  20,
                  24,
                  32,
                  48,
                  64,
                  96,
                  128,
                ],
                "space": Array [
                  0,
                  4,
                  8,
                  16,
                  32,
                  64,
                  128,
                  256,
                  512,
                ],
              }
            }
            w={
              Array [
                1,
                0.75,
              ]
            }
          >
            <Clean.div
              blacklist={
                Array [
                  "m",
                  "mt",
                  "mr",
                  "mb",
                  "ml",
                  "mx",
                  "my",
                  "p",
                  "pt",
                  "pr",
                  "pb",
                  "pl",
                  "px",
                  "py",
                  "width",
                  "w",
                  "fontSize",
                  "f",
                  "color",
                  "bg",
                  "fontFamily",
                  "font",
                  "textAlign",
                  "align",
                  "lineHeight",
                  "fontWeight",
                  "letterSpacing",
                  "display",
                  "maxWidth",
                  "minWidth",
                  "height",
                  "maxHeight",
                  "minHeight",
                  "size",
                  "ratio",
                  "alignItems",
                  "alignContent",
                  "justifyContent",
                  "justify",
                  "flexWrap",
                  "wrap",
                  "flexBasis",
                  "flexDirection",
                  "flex",
                  "justifySelf",
                  "alignSelf",
                  "order",
                  "gridGap",
                  "gridColumnGap",
                  "gridRowGap",
                  "gridColumn",
                  "gridRow",
                  "gridAutoFlow",
                  "gridAutoColumns",
                  "gridAutoRows",
                  "gridTemplateColumns",
                  "gridTemplateRows",
                  "border",
                  "borderTop",
                  "borderRight",
                  "borderBottom",
                  "borderLeft",
                  "borderColor",
                  "borderRadius",
                  "boxShadow",
                  "background",
                  "backgroundImage",
                  "bgImage",
                  "backgroundSize",
                  "bgSize",
                  "backgroundPosition",
                  "bgPosition",
                  "backgroundRepeat",
                  "bgRepeat",
                  "position",
                  "zIndex",
                  "top",
                  "right",
                  "bottom",
                  "left",
                  "hover",
                  "focus",
                  "active",
                  "disabledStyle",
                  "textStyle",
                  "colors",
                  "buttonStyle",
                  "borderWidth",
                ]
              }
              className="sc-bdVaJa biKVss"
              is="div"
              theme={
                Object {
                  "breakpoints": Array [
                    "40em",
                    "52em",
                    "64em",
                  ],
                  "fontSizes": Array [
                    12,
                    14,
                    16,
                    20,
                    24,
                    32,
                    48,
                    64,
                    96,
                    128,
                  ],
                  "space": Array [
                    0,
                    4,
                    8,
                    16,
                    32,
                    64,
                    128,
                    256,
                    512,
                  ],
                }
              }
              w={
                Array [
                  1,
                  0.75,
                ]
              }
            >
              <div
                className="sc-bdVaJa biKVss"
              >
                <ArticleItem
                  article={
                    Object {
                      "author": "Leandro Oriente",
                      "date": "2016-08-23T22:18:30.000Z",
                      "description": "Como chegar ao máximo de performance usando static generators e NGINX",
                      "html": null,
                      "lang": "pt-BR",
                      "slug": "criando-sites-estaticos-extremamente-performaticos",
                      "thumbnail": "/images/webperformance-thumb.png",
                      "title": "Criando sites estáticos extremamente performáticos",
                      "url": "/criando-sites-estaticos-extremamente-performaticos",
                    }
                  }
                  key="criando-sites-estaticos-extremamente-performaticos"
                >
                  <div
                    className="wrapper"
                  >
                    <p
                      className="title"
                    >
                      <Link
                        href="/criando-sites-estaticos-extremamente-performaticos"
                      >
                        <a
                          className="link"
                          href="/criando-sites-estaticos-extremamente-performaticos"
                          onClick={[Function]}
                        >
                          Criando sites estáticos extremamente performáticos
                        </a>
                      </Link>
                    </p>
                    <div>
                      <Badge
                        className="badge"
                      >
                        <span
                          className="badge badge"
                        >
                          August 2016
                        </span>
                      </Badge>
                      <Badge
                        className="badge"
                      >
                        <span
                          className="badge badge"
                        >
                          pt-BR
                        </span>
                      </Badge>
                    </div>
                  </div>
                </ArticleItem>
                <ArticleItem
                  article={
                    Object {
                      "author": "Leandro Oriente",
                      "date": "2014-07-29T00:00:00.000Z",
                      "description": "Uma dúvida recorrente que vejo em grupos ou que recebo por e-mail é sobre como cobrar por um freela. Particularmente não faço freelas tem muito tempo e não pretendo fazer mais, porém aprendi alguns métodos bem eficazes no período em que fazia e que funcionaram muito bem.",
                      "html": null,
                      "lang": "pt-BR",
                      "slug": "freelancer-como-cobrar-por-um-freelance",
                      "thumbnail": "/images/clientes-cobrar-thumb.jpg",
                      "title": "Como cobrar por um freelance",
                      "url": "/freelancer-como-cobrar-por-um-freelance",
                    }
                  }
                  key="freelancer-como-cobrar-por-um-freelance"
                >
                  <div
                    className="wrapper"
                  >
                    <p
                      className="title"
                    >
                      <Link
                        href="/freelancer-como-cobrar-por-um-freelance"
                      >
                        <a
                          className="link"
                          href="/freelancer-como-cobrar-por-um-freelance"
                          onClick={[Function]}
                        >
                          Como cobrar por um freelance
                        </a>
                      </Link>
                    </p>
                    <div>
                      <Badge
                        className="badge"
                      >
                        <span
                          className="badge badge"
                        >
                          July 2014
                        </span>
                      </Badge>
                      <Badge
                        className="badge"
                      >
                        <span
                          className="badge badge"
                        >
                          pt-BR
                        </span>
                      </Badge>
                    </div>
                  </div>
                </ArticleItem>
                <ArticleItem
                  article={
                    Object {
                      "author": "Leandro Oriente",
                      "date": "2014-03-09T00:00:00.000Z",
                      "description": "Tracei uma nova meta de vida. Aprender inglês de verdade em 1 ano. Tenho uma base razoavel a bastante tempo que me permite ler artigos, posts, documentações em inglês e compreender boa parte delas mas que não é o suficiente para conversar por 5 minutos com alguém fluente sem parecer um idiota.",
                      "html": null,
                      "lang": "pt-BR",
                      "slug": "como-aprender-ingles",
                      "thumbnail": "/images/joel-santana-thumb.jpg",
                      "title": "Como aprender inglês",
                      "url": "/como-aprender-ingles",
                    }
                  }
                  key="como-aprender-ingles"
                >
                  <div
                    className="wrapper"
                  >
                    <p
                      className="title"
                    >
                      <Link
                        href="/como-aprender-ingles"
                      >
                        <a
                          className="link"
                          href="/como-aprender-ingles"
                          onClick={[Function]}
                        >
                          Como aprender inglês
                        </a>
                      </Link>
                    </p>
                    <div>
                      <Badge
                        className="badge"
                      >
                        <span
                          className="badge badge"
                        >
                          March 2014
                        </span>
                      </Badge>
                      <Badge
                        className="badge"
                      >
                        <span
                          className="badge badge"
                        >
                          pt-BR
                        </span>
                      </Badge>
                    </div>
                  </div>
                </ArticleItem>
                <ArticleItem
                  article={
                    Object {
                      "author": "Leandro Oriente",
                      "date": "2014-01-19T00:00:00.000Z",
                      "description": "Não consigo clientes porque não tenho portfólio, não tenho portfólio porque não consigo clientes",
                      "html": null,
                      "lang": "pt-BR",
                      "slug": "nao-tenho-portfolio-porque-nao-tenho-clientes",
                      "thumbnail": "/images/nao-tenho-portfolio-thumb.jpg",
                      "title": "Não consigo clientes porque não tenho portfólio",
                      "url": "/nao-tenho-portfolio-porque-nao-tenho-clientes",
                    }
                  }
                  key="nao-tenho-portfolio-porque-nao-tenho-clientes"
                >
                  <div
                    className="wrapper"
                  >
                    <p
                      className="title"
                    >
                      <Link
                        href="/nao-tenho-portfolio-porque-nao-tenho-clientes"
                      >
                        <a
                          className="link"
                          href="/nao-tenho-portfolio-porque-nao-tenho-clientes"
                          onClick={[Function]}
                        >
                          Não consigo clientes porque não tenho portfólio
                        </a>
                      </Link>
                    </p>
                    <div>
                      <Badge
                        className="badge"
                      >
                        <span
                          className="badge badge"
                        >
                          January 2014
                        </span>
                      </Badge>
                      <Badge
                        className="badge"
                      >
                        <span
                          className="badge badge"
                        >
                          pt-BR
                        </span>
                      </Badge>
                    </div>
                  </div>
                </ArticleItem>
                <ArticleItem
                  article={
                    Object {
                      "author": "Leandro Oriente",
                      "date": "2014-01-12T00:00:00.000Z",
                      "description": "Atenção, esse é um post polêmico. Antes de iniciar o processo de leitura, esvazie a mente, procure um lugar tranquilo e preparesse porque a mensagem não é das mais agradáveis.",
                      "html": null,
                      "lang": "pt-BR",
                      "slug": "seu-cliente-precisa-de-um-site",
                      "thumbnail": "/images/cliente-nao-precisa-site-thumb.jpg",
                      "title": "Meu cliente realmente precisa de um site?",
                      "url": "/seu-cliente-precisa-de-um-site",
                    }
                  }
                  key="seu-cliente-precisa-de-um-site"
                >
                  <div
                    className="wrapper"
                  >
                    <p
                      className="title"
                    >
                      <Link
                        href="/seu-cliente-precisa-de-um-site"
                      >
                        <a
                          className="link"
                          href="/seu-cliente-precisa-de-um-site"
                          onClick={[Function]}
                        >
                          Meu cliente realmente precisa de um site?
                        </a>
                      </Link>
                    </p>
                    <div>
                      <Badge
                        className="badge"
                      >
                        <span
                          className="badge badge"
                        >
                          January 2014
                        </span>
                      </Badge>
                      <Badge
                        className="badge"
                      >
                        <span
                          className="badge badge"
                        >
                          pt-BR
                        </span>
                      </Badge>
                    </div>
                  </div>
                </ArticleItem>
                <ArticleItem
                  article={
                    Object {
                      "author": "Leandro Oriente",
                      "date": "2013-10-15T00:00:00.000Z",
                      "description": "O foco do post é atender a galera iniciante e encurtar alguns caminhos, porém se você é mais cascudo e quiser colaborar, fique a vontade.",
                      "html": null,
                      "lang": "pt-BR",
                      "slug": "guia-de-estudo-front-end",
                      "thumbnail": "/images/guia-comovente-thumb.jpg",
                      "title": "Um comovente guia de estudo para desenvolvedores front end",
                      "url": "/guia-de-estudo-front-end",
                    }
                  }
                  key="guia-de-estudo-front-end"
                >
                  <div
                    className="wrapper"
                  >
                    <p
                      className="title"
                    >
                      <Link
                        href="/guia-de-estudo-front-end"
                      >
                        <a
                          className="link"
                          href="/guia-de-estudo-front-end"
                          onClick={[Function]}
                        >
                          Um comovente guia de estudo para desenvolvedores front end
                        </a>
                      </Link>
                    </p>
                    <div>
                      <Badge
                        className="badge"
                      >
                        <span
                          className="badge badge"
                        >
                          October 2013
                        </span>
                      </Badge>
                      <Badge
                        className="badge"
                      >
                        <span
                          className="badge badge"
                        >
                          pt-BR
                        </span>
                      </Badge>
                    </div>
                  </div>
                </ArticleItem>
                <ArticleItem
                  article={
                    Object {
                      "author": "Leandro Oriente",
                      "date": "2013-08-21T00:00:00.000Z",
                      "description": "Minha visão de obrigações e diferenciais de desenvolvedores front end e algumas pequenas dicas a respeito.",
                      "html": null,
                      "lang": "pt-BR",
                      "slug": "como-me-tornar-um-desenvolvedor-front-end",
                      "thumbnail": "/images/developer-thumb.jpg",
                      "title": "Como me tornar um Desenvolvedor Front End",
                      "url": "/como-me-tornar-um-desenvolvedor-front-end",
                    }
                  }
                  key="como-me-tornar-um-desenvolvedor-front-end"
                >
                  <div
                    className="wrapper"
                  >
                    <p
                      className="title"
                    >
                      <Link
                        href="/como-me-tornar-um-desenvolvedor-front-end"
                      >
                        <a
                          className="link"
                          href="/como-me-tornar-um-desenvolvedor-front-end"
                          onClick={[Function]}
                        >
                          Como me tornar um Desenvolvedor Front End
                        </a>
                      </Link>
                    </p>
                    <div>
                      <Badge
                        className="badge"
                      >
                        <span
                          className="badge badge"
                        >
                          August 2013
                        </span>
                      </Badge>
                      <Badge
                        className="badge"
                      >
                        <span
                          className="badge badge"
                        >
                          pt-BR
                        </span>
                      </Badge>
                    </div>
                  </div>
                </ArticleItem>
                <ArticleItem
                  article={
                    Object {
                      "author": "Leandro Oriente",
                      "date": "2013-08-17T00:00:00.000Z",
                      "description": "Quero mostrar minha experiência atual com o mercado de trabalho e com o mundo acadêmico para depois apresentar as conclusões que tirei.",
                      "html": null,
                      "lang": "pt-BR",
                      "slug": "faculdade-de-informatica-fazer-ou-nao-fazer",
                      "thumbnail": "/images/classroom-thumb.jpg",
                      "title": "Faculdade de Informática. Fazer ou não fazer?",
                      "url": "/faculdade-de-informatica-fazer-ou-nao-fazer",
                    }
                  }
                  key="faculdade-de-informatica-fazer-ou-nao-fazer"
                >
                  <div
                    className="wrapper"
                  >
                    <p
                      className="title"
                    >
                      <Link
                        href="/faculdade-de-informatica-fazer-ou-nao-fazer"
                      >
                        <a
                          className="link"
                          href="/faculdade-de-informatica-fazer-ou-nao-fazer"
                          onClick={[Function]}
                        >
                          Faculdade de Informática. Fazer ou não fazer?
                        </a>
                      </Link>
                    </p>
                    <div>
                      <Badge
                        className="badge"
                      >
                        <span
                          className="badge badge"
                        >
                          August 2013
                        </span>
                      </Badge>
                      <Badge
                        className="badge"
                      >
                        <span
                          className="badge badge"
                        >
                          pt-BR
                        </span>
                      </Badge>
                    </div>
                  </div>
                </ArticleItem>
              </div>
            </Clean.div>
          </Box>
        </div>
      </Clean.div>
    </Flex>
  </section>
</ArticlesList>
`;
